### PR TITLE
Audioconsole and new music

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
See title. Also renames the jukebox from "space jukebox" to just "jukebox". Console sprites from Bay. Music is from Free Music Archive and PWMUS is from Bay. All are appropriately credited.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]
based.

File: code/game/machinery/jukebox.dm [hunk 1]
```diff
@@ -235,3 +235,29 @@ datum/track/New(var/title_name, var/audio)
...
 	if(playing)
 		add_overlay("[state_base]-running")
+
+/obj/machinery/media/jukebox/audioconsole
+	name = "audioconsole"
+	desc = "An Idris-designed jukebox for the 25th century. Unfortunately, someone made a mistake setting this one up - it isn't connected to the extranet - and only plays the demo music it was preprogrammed with."
+	icon = 'icons/obj/audioconsole.dmi'
+	icon_state = "audioconsole-nopower"
+	state_base = "audioconsole"
+	anchored = 0
```
Comment (SleepyGemmy) [inline code/game/machinery/jukebox.dm:245]:
```suggestion
	anchored = FALSE
```

File: code/game/machinery/jukebox.dm [hunk 2]
```diff
@@ -235,3 +235,29 @@ datum/track/New(var/title_name, var/audio)
...
+		new/datum/track("Phoron Will Make Us Rich", 'sound/music/audioconsole/PhoronWillMakeUsRich.ogg'),
+		new/datum/track("Amsterdam", 'sound/music/audioconsole/Amsterdam.ogg'),
+		new/datum/track("when", 'sound/music/audioconsole/When.ogg'),
+		new/datum/track("Vibe Ace", 'sound/music/audioconsole/VibeAce.ogg'),
+		new/datum/track("Backed Vibes Clean", 'sound/music/audioconsole/BackedVibesClean.ogg'),
+		new/datum/track("Lips", 'sound/music/audioconsole/Lips.ogg'),
+		new/datum/track("Childhood", 'sound/music/audioconsole/Childhood.ogg')
+	)
+
+/obj/machinery/media/jukebox/audioconsole/update_icon()
```
Comment (alsoandanswer) [inline code/game/machinery/jukebox.dm:259]: you added broken states, but didnt seem to include them here

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
It looks like there are a bunch of errant white pixels on the top left in the animations? Check them over for me, please.

**Comment by NonQueueingMatt** [comment]
immagine

**Comment by TheDanseMacabre** [comment]
> immagine
> 
> Same in the other animation.

I'm not really seeing these locally? Not sure why.

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**